Sekoia.io/sekoiaio/workspace/__init__.py

```diff
@@ -0,0 +1,14 @@
+from sekoia_automation.action import GenericAPIAction
+
+base_url = "api/v1/"
+
+
+GetCommunity = type(
```

Comment:
Out of curiosity what's the advantage of declaring the class this way instead of doing

```py
class GetCommunity(GenericAPIAction):
    verb = "get"
    endpoint =  base_url + "communities/{uuid}"
    query_parameters = []
```

Reply:
No idea, done out of consistency here :p